test/test_helper.exs

```diff
@@ -1,4 +1,8 @@
-File.rm_rf("test/tmp")
+tmp = Path.expand("test/tmp")
+
+File.rm_rf(tmp)
+File.mkdir_p(tmp)
```

Comment:
This change doesn't look related to the generalized time. Did you want it?

Reply:
I guess I could split this off into its own commit. It was a side effect of when I went to run the tests.